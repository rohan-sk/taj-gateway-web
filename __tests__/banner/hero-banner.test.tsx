import { render } from "@testing-library/react";
import HeroBanner from "../../components/banner/hero-banner.component";

const data = {
  image: "image-7d387f6df85235372783105d5f0db1d2d7d64bab-640x1307-png",
  largeImage: "image-6b856950349f3c128c7940e49d4023962f775597-1920x930-png",
};

it("shows hero banner", () => {
  const { asFragment } = render(<HeroBanner {...data} />);
  expect(asFragment()).toMatchSnapshot();
});

// describe("Home page components", () => {
//   it("renders a Header component", () => {
//     render(<Header {...data} />)
//   })

//   // it("should render without throwing an error", function () {
//   //   const wrapper = shallow(<Header {...data} />)
//   //   const test = wrapper.find(<AppBar />).length
//   //   expect(test).toEqual(1)
//   //   // expect(wrapper.find("p").text()).toBe("HOTELS")
//   // })
// })
