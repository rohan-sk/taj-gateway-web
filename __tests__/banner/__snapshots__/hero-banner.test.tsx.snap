// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows hero banner 1`] = `<DocumentFragment />`;
